install.sh

```diff
@@ -1,120 +1,149 @@
+    fedora) sudo dnf install -y make ;;
+    centos|rhel) sudo yum install -y make ;;
+    arch) sudo pacman -Sy make ;;
+    opensuse|suse) sudo zypper install -y make ;;
+    *) log "Unsupported Linux distribution. Please install make manually." $COLOR_RED; exit 1 ;;
+  esac
+  [ $? -eq 0 ] && log "make installed!" $COLOR_GREEN || { log "Failed to install make." $COLOR_RED; exit 1; }
 else
-  log "make already installed, skipping." 2
+  log "make already installed, skipping." $COLOR_GREEN
 fi
 
-log "\r\n#########################################################################" 4
-log "Checking Docker status" 4
+log "Checking Docker status..." $COLOR_CYAN
 if docker info >/dev/null 2>&1; then
-  log "Docker is running." 2
+  log "Docker is running." $COLOR_GREEN
 else
-  log "Docker is not running. Please run docker and try again." 1
-  log "You can run Docker service using: sudo systemctl start docker" 1
+  log "Docker is not running. Please run Docker and try again." $COLOR_RED
+  log "You can run Docker service using: sudo systemctl start docker" $COLOR_RED
   exit 1
 fi
 
-log "\r\n#########################################################################" 4
-log "Installing reNgine, please be patient it could take a while" 4
+log "Installing reNgine-ng, please be patient as it could take a while..." $COLOR_CYAN
+sleep 5
 
-log "\r\n=========================================================================" 5
-log "Generating certificates and building docker images" 5
-log "=========================================================================" 5
-make certs && make build && log "reNgine is built" 2 || { log "reNgine installation failed!!" 1; exit 1; }
+log "Generating certificates and building Docker images..." $COLOR_CYAN
+make certs && make build && log "reNgine-ng is built" $COLOR_GREEN || { log "reNgine-ng installation failed!" $COLOR_RED; exit 1; }
 
-log "\r\n=========================================================================" 5
-log "Docker containers starting, please wait celery container could be long" 5
-log "=========================================================================" 5
-make up && log "reNgine is installed!!!" 2 || { log "reNgine installation failed!!" 1; exit 1; }
+log "Docker containers starting, please wait as Celery container could take a while..." $COLOR_CYAN
+sleep 5
+make up && log "reNgine-ng is installed!" $COLOR_GREEN || { log "reNgine-ng installation failed!" $COLOR_RED; exit 1; }
 
-
-log "\r\n#########################################################################" 4
-log "Creating an account" 4
-log "#########################################################################" 4
+log "Creating an account..." $COLOR_CYAN
 make username
 
-log "\r\nThank you for installing reNgine, happy recon!!" 2
+log "\r\nThank you for installing reNgine-ng, happy recon!" $COLOR_GREEN
+
+log "\r\nIn case you're running this locally, reNgine-ng should be available at: https://127.0.0.1/" $COLOR_GREEN
```

Comment:
Could you catch the local ip ?
A one liner like this could
`ip -4 addr | grep 'state UP' -A2 | grep -o 'inet \([0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\)' | cut -d ' ' -f 2`

Reply:
I don't think this is reliable, as it doesn't print localhost for me at all (due to the `UNKNOWN` state). It could also output multiple IP-addresses. The thing is that you won't know which interface reNgine-ng/Docker is going to choose. In my case it outputs:

192.168.2.72, but reNgine-ng is running at 127.0.0.1. That one-liner doesn't output 127.0.0.1 because its state is UNKNOWN. This works and is shorter as well:

`ip -4 -br addr | awk '$2 == "UP" {print $3} /^lo/ {print $3}'`. This outputs the following for me:

```
127.0.0.1/8
192.168.2.72/24
```

Reply:
I made this more reliable now, see [my commit](https://github.com/Security-Tools-Alliance/rengine-ng/pull/95/commits/38c85ef443ad0cab1962194294062f2223f08196). Please test it and let me know if it works out well for you.

Reply:
Close to be good :grin: 
/24 or /8 need to be removed, because after install if you click on the link below in terminal
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/5a967bfa-a020-45a7-87c3-38da7a9efeb8)
It add a redirect on the login page
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/6c4b18fe-fc99-41da-aac7-f894f8d6b709)
Then after login it leads to a 500 error, which could be confusing
![image](https://github.com/Security-Tools-Alliance/rengine-ng/assets/1230954/1c3d1fcd-b3f4-4541-bbbc-f352ed6db9e2)

You could use this code to remove the bad part
```sh
url="https://172.16.0.184/24" 
clean_url=${url%/*}
echo "$clean_url"
```

Reply:
We also need to manage correctly the redirect, it should disply a 404 page, or redirect to homepage, not display a 500 error.
But it's another issue

Reply:
Yeah I also noticed this but didn't have this issue. It got cut automatically. Maybe browser difference. I'll remove it anyway because it looks more neat as well.

Reply:
Fixed: https://github.com/Security-Tools-Alliance/rengine-ng/pull/95/commits/c5e305e42cbb4865afe9e44c411f4329356dc54c